feature-extensions/apigee/apigee-installer/sharedflows/DataEncryption/sharedflowbundle/DataEncryption.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Speaking of zipfiles, can't you bundle all the shared flow stuff into a zip and deploy it as one unit?

I'm asking, I'm not sure of what the Apigee Way is.

Reply:
Each sharedflow is a deployable unit and hence cannot be bundled into one zip.

They are generic sharedflows and some of them are customized for Jazz use.